js/index.js

```diff
@@ -1,7 +1,75 @@
 // Iteration 1: Names and Input
+let hacker1 = "Pablo"; 
+console.log(`"The driver's name is ${hacker1} "`);
 
+let hacker2 = "Monica";
+console.log(`"The navigator's name is ${hacker2}"`);
 
 // Iteration 2: Conditionals
+let hackers1Long = hackers1.length;
+let hackers2Long = hackers2.length;
 
+if (hackers1Long > hackers2Long) {
+  console.log(`The driver has the longest name, it has ${hackers1Long} characters`)
+} else {
+  console.log(`It seems that the navigator has the longest name, it has ${hackers2Long} characters`)
+}
 
-// Iteration 3: Loops
\ No newline at end of file
+// Iteration 3: Loops
+//3.1 Print all the characters of the driver's name, separated by a space and in capitals i.e. "J O H N"
+for (let i = 0; i < hackers1.length; i++ ){
+    console.log(hackers1[i].toUpperCase());
+   }
```

Comment:
Lo mismo aquí: `ReferenceError: hackers1 is not defined`
La lógica es buena, pero fijate que imprime las letras en distintas líneas. Eso es porque en cada bucle, hace un console.log() que se imprime en lineas sucesivas. Si quieres evitar eso, puedes guardar la letra que obtienes en cada bucle en una cadena, y cuando acaba el bucle, hacer un único console.log()